05.ls/ls.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+MAX_COLUMNS = 3
+
+def main
+  all_files = if ARGV.empty?
+                Dir.glob('*')
+              else
+                other_directory_files = []
+                ARGV.each do |str|
+                  other_directory_files = Dir.glob("#{str}/*")
+                end
+                other_directory_files
+              end
```

Comment:
良い感じだとは思うのですが、このifの書き方をするとインデントがかなり深い部分にコードを書くことになりちょっと読みづらくなるかなと個人的には思っています。もう少し短ければいいですが、今回だと`else`ブロックが結構長いですし、この辺の処理を丸っとメソッドにしてあげた方が読みやすくなるかなと思いました。こんな感じで↓

```suggestion
  # メソッド名は適当です
  all_files = a_new_method(ARGV)
```